R/acuityview_pad.R

```diff
@@ -35,11 +32,13 @@ acuityview_pad <- function(image, obj_dist, obj_width, eye_res) {
 
   # Create an MTF matrix with dimensions equal to the image
   MTF <- matrix(NA, nrow = dim(image_pad)[2], ncol = dim(image_pad)[1])
+
+  # FIXME: This can properly be entirely vectorized without loops (nor *apply())
   for (i in 1:width_pix) {
     for (j in 1:width_pix) {
       x <- i - center
       y <- j - center
-      freq <- round(sqrt(x^2 + y^2)) / width_pix * (width_pix / width_deg)
+      freq <- round(sqrt(x^2 + y^2)) / width_deg
```

Comment:
This doesn't seem right. We need to double check the equations before merging.

Reply:
This is what they use in the original code:

https://github.com/eleanorcaves/AcuityView/blob/b0465ec9ff20eb5e1c2c6c15d13d08b680282f75/R/AcuityView.R#L77-L88

@thomased, I seem to remember that you are in touch with the original authors. Am I missing something or is it worth getting in touch to ask if the code is correct?